### DISCUSSION TITLE
Daphne will not launch

### DISCUSSION THREAD

**Discussion by Borfustic** [rg35xx-cfw/Koriki#73] [Q&A]
I followed every guide with reference to Batocera version for the RG35XX with regard to the laserdisc emaulator for the RG35XX.  

The games show up on the menu, yet fail to launch. 

The format for Daphne is similar to Retropie and follows the same naming convention for roms, etc. 

Is the reason for the failure due a bug?

There is very little information in relation to the port.

Thank you.

**Comment by benni092** [comment]
There is currently no daphne core in retroarch. Koriki is set to daphne_libretro, but there is no daphne_libretro.so in /usr/lib/libretro

**Reply by acmeplus** [reply]
Yes, there's no Daphne/hypseus core in koriki, but the standalone version works. However you need to have the right format for the "roms" to work with the standalone version. It also requires a dedicated custom launcher.

You can try to experiment with this script in the ports section (/userdata/roms/ports/daphne.sh):
```bash
#!/bin/sh

cd /usr/share/daphne

hypseus lair vldp -framefile /userdata/roms/daphne/framefile/lair.txt -homedir /userdata/roms/daphne -fastboot -fullscreen
```
That assumes that you have your Daphne roms in /userdata/roms/daphne, and that there's a framefile folder that includes lair.txt. The content of that lair.txt is typically like:
```bash
lair/

151     lair.m2v
```
so Daphne will expand the path and look for ``/userdata/roms/daphne/framefile/lair/lair.m2v`` for the game video files.
